crates/ruff_db/src/diagnostic.rs

```diff
@@ -0,0 +1,175 @@
+use ruff_text_size::TextRange;
+use salsa::Accumulator as _;
```

Comment:
Both work, for as long as there's no other type named `Accumulator`. Using `as _` imports that type "unnamed". You can't reference the type itself. That's often sufficient for traits because all that is needed is to bring the trait into the scope to call that trait's functions.